.github/workflows/tag-and-release.yaml

```
@@ -0,0 +1,8 @@
+name: Tag and Release
+on:
+  - issues
+  - pull_request
+jobs:
+  label-check:
```

actionlint

"steps" section is missing in job "label-check"

(syntax-check)